<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>guiWidget</class>
 <widget class="QWidget" name="guiWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>guiWidget</string>
  </property>
  <widget class="QListWidget" name="itemList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>256</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="itemComboBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>200</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>add item</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>60</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>item name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>90</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>item category:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>120</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>item price:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>150</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>offer list: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="itemNameLineEdit">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>60</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="categoryLineEdit">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>90</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="itemPriceLineEdit">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>120</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="offerListLineEdit">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>150</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="offerListWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>371</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>BID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bidLineEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bidButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Bid</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
